airflow/constraints-2.6.1-python39.txt

```diff
@@ -313,7 +313,7 @@ google-resumable-media==2.5.0
 googleapis-common-protos==1.56.4
 graphql-core==3.2.3
 graphviz==0.20.1
-greenlet==2.0.2
+greenlet==3.0.2
```

Comment:
I'm no Python guy but I assume we need to be careful here making a breaking change with a Airflow dependency.
https://github.com/python-greenlet/greenlet/blob/master/CHANGES.rst#300a1-2023-06-21 sounds like Python 2.12 addition and 2.7, 3.5 and 3.6 removal as well as C++ bump.

WDYT of pulling in `3.0.2` only for the experimental ARM builds to not disturb the "production" x84 image?